components/dashboard/sections/accounts/InternalTransferModal.tsx

```diff
@@ -85,12 +105,12 @@ export default function InternalTransferModal({
             toAccount: undefined,
             amount: {
               valueInCents: 0,
-              currency: accounts[0]?.currency || Currency.USD,
+              currency: activeAccounts[0]?.currency || Currency.USD,
```

Comment:
There's a bug here that causes the modal to be stuck in USD, even if the currency is EUR:
![image](https://github.com/user-attachments/assets/ccee20da-238e-499b-a20b-de3ee5838825)

To reproduce:
1. Refresh page;
2. Open Modal.

Reply:
My personal opinion: I feel like modals should be rendered only when being used and not hanging in a context somewhere to be displayed or not. This complexity is extended into the state management of the modal and its internal query. The performance gains are not worth the complexity in code and development.

Reply:
Thanks for catching this! Fixed now.